src/Commands/ConfigTrait.php

```diff
@@ -16,7 +16,7 @@ trait ConfigTrait
     protected function configurePPMOptions(Command $command)
     {
         $command
-            ->addOption('bridge', null, InputOption::VALUE_REQUIRED, 'Bridge for converting React Psr7 requests to target framework.', 'HttpKernel')
+            ->addOption('bridge', null, InputOption::VALUE_REQUIRED, 'Bridge for converting React Psr7 requests to target framework. (e.g., HttpKernel)', 'StaticBridge')
```

Comment:
I just want to point out that this warrants to major version bump for me. Changing the default behavior when this option is not provided has the potential to break existing setups.

Reply:
@jkrzefski you're totally right, although it sounds weird that someone would start ppm with the defaults for anything serious, logging and debugging are also on by default

Reply:
I can totally imagine people only setting those options whose default values don't match their use case. So debug behavior and logging might be disabled explicitly but the bridge selection is left unchanged, since it "works by default".